piwebapi_samples/Data Analysis/UOM-Ft3Min.xml

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This custom UOM is not loaded by the PIUploadUtility and is not mentioned in the ReadMe. It's a non-standard UOM so attempting to load the BuildingExample.xml without it will fail.

Reply:
I will add a comment to README

Reply:
Is it really necessary to include a custom UOM for this sample? Is there any way we could avoid it? This would likely never get cleaned up, would hate to dirty up customer UOM databases.